cmd/shove/main.go

```diff
@@ -150,9 +152,31 @@ func main() {
 			slog.Error("Failed to add WebPush service", "error", err)
 			os.Exit(1)
 		}
-	}
-
-	if *telegramBotToken != "" {
+	} else if *webPushVAPIDKeysJSON != "" {
+		content, err := os.ReadFile(*webPushVAPIDKeysJSON)
+		if err != nil {
+			slog.Error("Failed to read WebPush VAPID keys file", "error", err)
+			os.Exit(1)
+		}
+		var vapidKeys struct {
+			PublicKey  string `json:"publicKey"`
+			PrivateKey string `json:"privateKey"`
+		}
+		err = json.Unmarshal(content, &vapidKeys)
+		if err != nil {
+			slog.Error("Failed to unmarshal WebPush VAPID keys file, please check your JSON file containt 'publicKey' and 'privatekey' keys", "error", err)
+			os.Exit(1)
+		}
+		web, err := webpush.NewWebPush(vapidKeys.PublicKey, vapidKeys.PrivateKey, newServiceLogger("webpush"))
+		if err != nil {
+			slog.Error("Failed to setup WebPush service", "error", err)
+			os.Exit(1)
+		}
+		if err := s.AddService(web, *webPushWorkers, services.SquashConfig{}); err != nil {
+			slog.Error("Failed to add WebPush service", "error", err)
+			os.Exit(1)
+		}
+	} else if *telegramBotToken != "" {
```

Comment:
**Issue**: This should be an `else if` -- Telegram is independent from the webpush config.